Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,422 @@
+            }
+            return index;
+        }
+
+        /// <summary>
+        /// Return index of Min value in decimal array.
+        /// </summary>
+        private static int FindDecimalArrayMin(decimal[] inputArray, int startIndex = 0)
+        {
+            var min = inputArray[startIndex];
+            var index = startIndex;
+            for (int i = startIndex; i < inputArray.Length; i++)
+            {
+                if (inputArray[i] < min)
+                {
+                    min = inputArray[i];
+                    index = i;
+                }
+            }
+            return index;
+        }
+
+        /// <summary>
+        /// Returns an formated output of currency (ex. -¤1, ¤4, ¤1002, -¤1001).
+        /// </summary>
+        private static string FormatCurrency(decimal currency, string currencySymbol)
+        {
+            const char negativeSymbol = '-';
+
+            var formatedCurrency = new StringBuilder();
+
+            if (currency < 0)
+            {
+                currency *= -1;
+                formatedCurrency.Append(negativeSymbol).Append(currencySymbol).Append(currency);
+                return formatedCurrency.ToString();
+            }
+
+            formatedCurrency.Append(currencySymbol).Append(currency);
+            return formatedCurrency.ToString();
+        }
+
+        /// <summary>
+        /// Return formated output for multiple names in array (Kai, EarLington and Mihail).
+        /// </summary>
+        private static string FormatStringAndCommas(string[] validPeople)
+        {
+            var sb = new StringBuilder();
+
+            for (int i = 0; i < validPeople.Length; i++)
+            {
+                sb.Append(validPeople[i]);
+                if (i + 2 < validPeople.Length)
+                {
+                    sb.Append(", ");
+                }
+                else if (i + 1 < validPeople.Length)
+                {
+                    sb.Append(" and ");
+                }
+            }
+
+            return sb.ToString();
+        }
+
+        /// <summary>
+        /// Finds highest balance for a single person.
+        /// </summary>
+        private static decimal HighestBalanceForSinglePerson(string personAndBalance)
+        {
+            var balanceList = ConvertStringToDecimalArray(personAndBalance);
+            var highestBalanceIndex = FindDecimalArrayMax(balanceList);
+            var highestBalance = balanceList[highestBalanceIndex];
+
+            return highestBalance;
+        }
+
+        /// <summary>
+        /// Verifies if string is valid.
+        /// </summary>
+        private static bool InputStringIsValid(string inputString)
+        {
+            return !string.IsNullOrEmpty(inputString) && !string.IsNullOrWhiteSpace(inputString);
+        }
+
+        /// <summary>
+        /// Finds lowest balance for a single person.
+        /// </summary>
+        private static decimal LowestBalanceForSinglePerson(string personAndBalance)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'LowestBalanceForSinglePerson'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4970157)

Reply:
Fixed in [2db6c28](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/161/commits/2db6c282219933cc661146c542a210b0f73f44cf)